internal/pkg/variable/parser/parser.go

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) HashiCorp, Inc.
+// SPDX-License-Identifier: MPL-2.0
+
+package parser
+
+import (
+	"github.com/hashicorp/hcl/v2"
+	"github.com/hashicorp/nomad-pack/internal/pkg/variable/parser/config"
+)
+
+type Parser interface {
```

Comment:
sooo we have `internal/pkg/variable/parser.go:Parser` and `internal/pkg/variable/parser/parser.go:Parser` interfaces? 🤔 
Also my comment from the previous `Parser` interface definition above still applies here.